### PR TITLE
Improve understandability of Settingsscreen.

### PR DESCRIPTION
**Pull request by jessearens**
Now the user does not see the internally used value, but the description of the values they can change.

<user_interface

### CODE REVIEW

**Comment by ChielBruin** [comment]
Nice change! The settings are much more comprehensive now!
<user-interface

File: src/main/java/com/github/fishio/control/SettingsScreenController.java [hunk 1]
```diff
@@ -52,7 +52,7 @@ public void onSwitchTo() {
 	 */
 	private int showSliderSettings(int row) {
 		for (String key : settings.getSliderSettings()) {
-			Label label = new Label(key);
+			Label label = new Label(settings.getDescription(key));
```
Comment (ChielBruin) [inline src/main/java/com/github/fishio/control/SettingsScreenController.java:55]: You missed the tooltip (that was missing)
<review

**Comment by Diocruel** [comment]
Nice changes, though implement the note Chiel made.

<review

**Comment by jessearens** [comment]
Added the tooltip for music settings.

<feature,user_interface

**Comment by Diocruel** [comment]
Okay, ready for merging then!

<review

**Pull request merged**